app/src/main/java/org/wikipedia/analytics/metricsplatform/ApplicationData.kt

```diff
@@ -0,0 +1,15 @@
+package org.wikipedia.analytics.metricsplatform
+
+import org.wikipedia.WikipediaApp
+
+object ApplicationData {
+
+    val data = mapOf(
```

Comment:
This field is statically initialized with the values of the app's theme and language codes at the time of initialization, when in fact those values can change throughout the lifetime of the app. (meaning that this field won't always contain the current values.)

To make this field _dynamically_ return a fresh map every time, we can use the Kotlin syntax of a getter method:
```
val data get() = mapOf(...
```

Reply:
nice -- assuming then by leveraging the getter method, everything included here will be fetched dynamically